lib/vanagon/engine/always_be_scheduling.rb

```diff
@@ -37,7 +37,21 @@ def name
 
       # return the platform name as the "host" name
       def build_host_name
-        @platform.name
+        platform_name_to_vmpooler_style(@platform.name)
+      end
+
+      def platform_name_to_vmpooler_style(platform)
+        translation = {
+          "aix-5.3-ppc" => "aix-53-ppc",
```

Comment:
can this be done generically? When AIX 7.2 or 8 comes out, it'd be pretty cool to not have to remember this is hard-coded in vanagon somewhere.